terraso_backend/apps/soil_id/models/soil_data.py

```diff
@@ -13,11 +13,24 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see https://www.gnu.org/licenses/.
 
+from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 from apps.core.models.commons import BaseModel
 from apps.project_management.models.sites import Site
 
+CONCAVE = "concave"
+CONVEX = "convex"
+LINEAR = "linear"
+
+SLOPE_SHAPES = ((CONCAVE, _("Concave")), (CONVEX, _("Convex")), (LINEAR, _("Linear")))
+
 
 class SoilData(BaseModel):
     site = models.OneToOneField(Site, on_delete=models.CASCADE)
+    slope = models.IntegerField(
+        null=True, validators=[MinValueValidator(0), MaxValueValidator(100)]
+    )
+    slope_shape = models.CharField(null=True, choices=SLOPE_SHAPES)
+    # add slope aspect ?? (north, south, east ..)
```

Comment:
@ltseng what is the status of the slope aspect and landscape position data inputs? the product spec for them still has all that info as a suggestion, are those still in flux and we should hold off looking at them for now or can we implement according to those suggestions?

Reply:
Still in flux, but I don't think I'm going to be able to dissuade BLM on those two. You can proceed with this current PR for now, and I'll file a separate issue for additions when that's ready.

Reply:
@ltseng Just confirming, for this PR I won't include slope aspect and landscape position data inputs, is that right?

Reply:
Right now, no. I'll write a different issue for slope aspect and landscape position later!